apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue

```diff
@@ -12,17 +12,108 @@
                 </p-button>
             </template>
         </p-heading>
+        <!-- TODO: bind other props of PToolboxTable -->
+        <p-toolbox-table :fields="fields"
+                         :items="state.secrets"
+        />
     </p-pane-layout>
 </template>
 
 <script lang="ts" setup>
-import { defineProps } from 'vue';
+import {
+    defineProps, reactive, watch, onMounted, computed,
+} from 'vue';
 
-import { PHeading, PButton, PPaneLayout } from '@spaceone/design-system';
+import {
+    PHeading, PButton, PPaneLayout, PToolboxTable,
+} from '@spaceone/design-system';
+import type { DefinitionField } from '@spaceone/design-system/types/data-display/tables/definition-table/type';
 
+import { store } from '@/store';
 import { i18n } from '@/translations';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars,@typescript-eslint/ban-types
-const props = defineProps<{}>();
+import type { ServiceAccountReferenceMap } from '@/store/modules/reference/service-account/type';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+interface SecretModel {
+    secret_id: string;
+    provider: string;
+    service_account_id: string;
+    project_id: string;
+    created_at: string;
+}
```

Comment:
suggestions: We were managing types in a separate file, is there a specific guideline for that?

Reply:
Oh this is temporary type declaration! I forgot to left a comment! Sorry ~